frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -238,6 +238,35 @@ i.e.
   padding: units(1) units(2);
 }
 
+.simpler-save-button {
+  padding: units(0.5) units(2) units(0.5) units(1);
```

Comment:
can we do this with existing classes?

Reply:
Let's discuss with @andycochran Since we are creating our own components I think it makes sense to use our own css. I'd like to stick with this for now.